components/text/Paragraph.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import Base, { BaseProps } from './Base';
+
+interface ParagraphProps extends BaseProps {}
+
+const Paragraph: React.SFC<ParagraphProps> = (props: ParagraphProps) => (
```

Comment:
`(props: ParagraphProps)` 这里的类型可以省略。

```suggestion
const Paragraph: React.SFC<ParagraphProps> = (props) => (
```